.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -52,76 +20,42 @@ body:
 
     - type: textarea
       attributes:
-          label: Expected Behavior
-          description: Please describe what you expected to happen.
+          label: Expected vs Current Behavior
+          description: Please describe what you expected to happen and what you are actually experiencing.
```

Comment:
I'd change this to "Please describe what you expect to happen and what is currently happening."